src/SOS/Strike/strike.cpp

```diff
@@ -13709,8 +13709,21 @@ class ClrStackImpl
             if (bDisplayRegVals)
                 PrintManagedFrameContext(pStackWalk);
 
-        } while (pStackWalk->Next() == S_OK);
+            hr = pStackWalk->Next();
+        } while (hr == S_OK);
 
+        if (FAILED(hr))
+        {
+            // Normal stack walk ends with S_FALSE
+            // Failure means the stalk walk did not complete normally
+            ExtOut("<failed>\nStack Walk failed. Reported stack incomplete.\n");
```

Comment:
Printing the hr after "failed." may be useful. Like "... failed 0x%08x."

Reply:
I can add it if you want.  At the moment all failures in `stackWalk->Next()` report E_FAIL.

Reply:
Don't worry about it then.